examples/pylab_examples/usetex_demo.py

```diff
@@ -1,70 +1,70 @@
+import matplotlib.pyplot as plt
 import matplotlib
 matplotlib.rc('text', usetex=True)
-import pylab
```

Comment:
Can you move the import of pyplot to after matplotlib.rc('text', usetex=True)

The way matplotlib works it is best to modify the rc before importing pyplot